.circleci/config.yml

```diff
@@ -122,6 +122,18 @@ commands:
           name: Pip-compile requirements file
           command: make pip-compile
 
+  deploy_viz:
```

Comment:
This isn't really deploying viz though is it? I think `run_viz_build` might be a more suitable name.

Reply:
Good call. Will change.